tests/helpers.py

```diff
@@ -90,3 +90,11 @@ def send(self, *args):
 
 
 _MockResponse = namedtuple('MockResponse', ['status_code', 'text', 'headers'])
+
+
+def assert_raises(exception_class, action):
+  try:
+    action()
+    raise AssertionError, "Should have raised an %s" % exception_class
```

Comment:
If another type of exception is thrown it will be uncaught, meaning that the test will fail. And you get more detailed info than just AssertionError("Some other error was thrown.")